versions/build.gradle.kts

```diff
@@ -150,6 +150,10 @@ dependencies {
 
     include(libs.caffeine, relocate = true)
 
+    // TODO: Remove
+    implementation("com.github.JnCrMx:discord-game-sdk4j:0.5.5")
+    implementation("com.github.caoimhebyrne:KDiscordIPC:0.2.1")
+
     // for other mods and universalcraft
```

Comment:
Please do not use 2 rpc libs, you also have to use include instead of implementation here for it to work outside of dev env